lib/modules/files-secrets/config-schema.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "Schema of Hawkeye module files-secrets",
+  "properties": {
+    "files": {
+      "type": "array",
+      "title": "List of files containing patterns of suspicious filenames",
+      "default": [],
+      "items": {
+        "type": "string",
+        "title": "Path to file",
+        "examples": [
+          "path/to/patterns.json"
+        ]
+      }
+    }
+  },
+  "examples": [
+    {
+      "files": [
+        "path/to/patterns.json"
+      ]
+    }
+  ]
+}
```

Comment:
To generate reasonable documentation (help) we can enforce by tests things like:
  * each config should have an example (that is up to date and passes schema test)
  * each property has a title and a type

Reply:
Also we can try to use this definition for cli arguments. Not sure how hard it will be if we want to make it good

Reply:
... or limit cli arguments to what we have now and allow to pass `.hawkeyerc` as an argument or even read it from stdIn